### DISCUSSION TITLE
Copilot Plugin doesn't work in pycharm professional inside jupyter notebooks

### DISCUSSION THREAD

**Discussion by data-stepper** [community/community#36668] [Copilot Conversations]
I'm using Version 1.1.33.1980 of the copilot plugin (updated pycharm professional and the plugin just now) and it works fine in normal .py files (and so on) but not inside jupyter notebooks. When opening the completions tab, it says `An error occurred while fetching the completions. Please try again`.

The plugin works for jupyter notebooks inside vs code just fine.

I hope there is a fix for this since I love working with pycharm and jupyter notebooks.

Thanks in advance

**Comment by iamcjwilliams** [comment]
@data-stepper Thank you for your feedback, our copilot team is investigating this issue

**Comment by ossanna16** [comment]
Hey there @data-stepper and thank you for your report! I'm glad you found our community 🙂
